### PR TITLE
Nest spinnaker elephant demo

### PR DESCRIPTION
**Pull request by BerndSchuller**
create result bundle, in order to copy results back to collab

### CODE REVIEW

File: Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def cch_vista_submit_task(inputdata_spinnaker, inputdata_nest, run_script,
                 description: Number of tasks which will be run on the HPC.
 
         Returns:
-            res: application/unknown
+            res: application/vnd.juelich.bundle.elephant.data
```
Comment (alperyeg) [inline Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py]: this must be registered right? Why not just use the h5 file as return value?

Comment (BerndSchuller) [inline Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py]: I don't know - feel free to change it :-)

Comment (alperyeg) [inline Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py]: I mean we do not produce bundles. It is just a h5 file we write back.

Comment (BerndSchuller) [inline Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py]: ok

Comment (alperyeg) [inline Elephant/pairwise_correlation_histogram_task/cch_vista_submit_task.py]: Then I am merging this and making a new PR.

**Pull request merged**